lib/litani_report.py

```diff
@@ -547,9 +547,12 @@ def render(run, report_dir):
 
     pipe_templ = env.get_template("pipeline.jinja.html")
     for pipe in run["pipelines"]:
-        PipelineDepgraphRenderer.render(
-            render_root=temporary_report_dir,
-            pipe_url=pathlib.Path(pipe["url"]), pipe=pipe)
+        if pipeline_degraph_renderer:
```

Comment:
I still don't like doing this check here. It seems cleaner to me to pass in a PipelineDepgraphRenderer no matter what (so never pass None), but the `render` method of the object itself should just be a no-op if dot is not installed.

For example, the __init__ method of PipelineDepgraphRenderer could run `which`, and if dot isn't installed it would set an instance variable to False, and then the render method would check that variable and return early.